src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,38 +1,33 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
-
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
```

Comment:
should we use nullable = false? and name = "id"?

Reply:
no for id, because they are repeating the same name of the variable it will be like Comment_id_id.
nullable for id with @GeneratedValue we do not need it as well. But we do need them when we want to restrict the columns to be not null. For this place it is redundant :)